core/payment-driver/zksync/src/zksync/mod.rs

```diff
@@ -0,0 +1,9 @@
+/*
+    Private mod to encapsulate all zksync logic, revealed from the `wallet`.
+*/
+
+pub mod wallet;
+
+mod faucet;
```

Comment:
Not quite sure if `faucet` belongs here but I'm not going to push it.

Reply:
why not? Just curious :)

Reply:
Well, I have this feeling like it's not really part of the zkSync infrastructure but and external service. But it's tightly coupled, so it's fine.